powerbi-docs/developer/embedded/move-to-production.md

```diff
@@ -56,19 +56,17 @@ If you run out of embed tokens while testing, you need to purchase a Power BI Em
 
 Once you create a capacity, you can assign your workspace to that capacity.
 
-All the workspaces that contain Power BI resources related to the embedded content (including datasets, reports, and dashboards), must be assigned to capacities. For example, if an embedded report and the dataset bound to it reside in different workspaces, both workspaces must be assigned to capacities.
+Each workspace that contains a Power BI artifact related to the embedded content (including datasets, reports, and dashboards) must be assigned to capacities. For example, if an embedded report and the dataset bound to it reside in different workspaces, both workspaces must be assigned to capacities.
```

Comment:
We try not to use the word 'artifact' - let's stick with the original sentence.

Reply:
Ah interesting. I was looking for a better word than "resource" to distinguish it from the embedded resource that you link the workspace to (which is referred to twice on this page), and I thought I nailed it because "artifact" has been used in other pages. In that case, is there another word we could use here instead of "resource"?